src/common/auth.middleware.ts

```diff
@@ -8,14 +8,17 @@ export class AuthMiddleware implements IMiddleware {
 	constructor(private secret: Secret) {}
 
 	execute(req: Request, res: Response, next: NextFunction): void {
-		let token = req.body.token || req.headers.authorization;
+		let token = req.body.token || req.headers.authorization || req.query.token;
 
 		if (!token) {
 			return next();
 		}
 
-		try {
+		if (token.split(' ')[0] === 'Bearer') {
```

Comment:
I think this conditions is not required
you will add Bearer prefix in any case
let's remove condition and leave only this string
```js
token = token.split(' ')[1];
```

Reply:
In req.query.token token comes without "Bearer " prefix right now. In the future, when activation request will arrive from client this behavior must be changed and this check of conditions will be removed.

Reply:
Unified for waiting for activation token in req.body.activationToken